services/static-webserver/client/source/class/osparc/desktop/account/ProfilePage.js

```diff
@@ -215,15 +228,27 @@ qx.Class.define("osparc.desktop.account.ProfilePage", {
     },
 
     __createPrivacySection: function() {
+      // binding to a model
+      const raw = {
+        "hideUsername": false,
+        "hideFullname": true,
+        "hideEmail": true,
+      };
+
+      const privacyModel = this.__userPrivacyModel = qx.data.marshal.Json.createModel(raw);
+
       const box = osparc.ui.window.TabbedView.createSectionBox(this.tr("Privacy"));
       box.set({
         alignX: "left",
         maxWidth: 500
       });
 
-      const label = osparc.ui.window.TabbedView.createHelpLabel(this.tr("For Privacy reasons, you might want to hide your First and Last Names and/or the Email to other users"));
+      const label = osparc.ui.window.TabbedView.createHelpLabel(this.tr("For Privacy reasons, you might want to hide some personal data."));
```

Comment:
Q: since all your messages are wrapped on `this.tr`, can you dump them in a file. Then we can run chatty against it with all the rules we have and put them back

Reply:
Yes, I can